integreat_cms/cms/views/external_calendars/external_calendar_form_view.py

```diff
@@ -139,3 +121,45 @@ def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
                 "current_menu_item": "external_calendar_list",
             },
         )
+
+    def show_import_messages(
+        self, external_calendar_instance: ExternalCalendar, import_result: ImportResult
+    ) -> None:
+        """
+        This function shows the message after the import has been done
+
+        :param external_calendar_instance: The instance of the external calendar
+        :param import_result: The results of the import, either successes or errors
+        """
+        if external_calendar_instance is not None:
+            number_of_errors = import_result.number_of_errors
+            number_of_successes = external_calendar_instance.events.count()
+        else:
+            number_of_errors = 0
+            number_of_successes = 0
```

Comment:
```suggestion
```

Isn't this code unreachable, because the external calendar was previously already used in `import_events`?

Reply:
Oh yes, you're right. This is still a leftover of the previous version. I will remove it :)